Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserWidget.cpp

```diff
@@ -696,8 +696,8 @@ void ezQtAssetBrowserWidget::OnTransform()
     ezUuid guid = m_pModel->data(index, ezQtAssetBrowserModel::UserRoles::AssetGuid).value<ezUuid>();
     QString sPath = m_pModel->data(index, ezQtAssetBrowserModel::UserRoles::RelativePath).toString();
     range.BeginNextStep(sPath.toUtf8());
-    auto res = ezAssetCurator::GetSingleton()->TransformAsset(guid, ezTransformFlags::TriggeredManually);
-    if (res.m_Result.Failed())
+    ezTransformStatus res = ezAssetCurator::GetSingleton()->TransformAsset(guid, ezTransformFlags::BackgroundProcessing);
```

Comment:
Is this change intentional ?

Reply:
First I tought I'd have to change it in order to make this consistent with the backgroundprocessing but it will only make things slower as we will do the same operation twice on the same thread for no good reason. Good catch.